fastlane/lib/fastlane/actions/oclint.rb

```diff
@@ -23,7 +23,11 @@ def self.run(params)
         select_regex = params[:select_regex] if params[:select_regex] # Overwrite deprecated select_reqex
         exclude_regex = params[:exclude_regex]
 
-        files = JSON.parse(File.read(compile_commands)).map { |compile_command| compile_command['file'] }
+        files = JSON.parse(File.read(compile_commands)).map do |compile_command|
+          file = compile_command['file']
+          File.exist?(file) ? file : compile_command['directory'] + '/' + file
```

Comment:
we make other commit that fit hound violation `expect(result).to match(%r{cd .* && oclint -report-type=html -o=oclint_report.html -p .*?compile_commands\.json \".*})` and your suggestion after all will be green green maybe you prefer we create a new PR to have a clean commit history instead tentative?